pkg/controller/clusterstorage/clusterstorage_controller.go

```diff
@@ -183,7 +184,36 @@ func (r *ReconcileClusterStorage) Reconcile(request reconcile.Request) (reconcil
 		return reconcile.Result{}, err
 	}
 
-	// StorageClass already exists - don't requeue
+	// Check to see if modifications have been made to the StorageClass attributes
+	needUpdate := false
+	comparisonSC := sc.DeepCopy()
+	comparisonSC.ObjectMeta = found.ObjectMeta
```

Comment:
nit: perhaps `[old|existing]SC` and `newSC` would be a better representation for the storageClasses?

Reply:
I went through the method and updated the variables names as follows:

* found - has now been set to `existingSC`
* sc - has now been set to `newSCFromFile`